tests/docker_common.py

```diff
@@ -59,21 +60,26 @@ def diff_dict(left, right):
 
 def event_test(agent, name, pre_func=None, post_func=None, diff=True):
     req = json_data(name)
-    resp_valid = json_data(name + '_resp')
+    valid_resp_file = json_data(name + '_resp')
+    valid_resp = JsonObject.unwrap(valid_resp_file)
 
     if pre_func is not None:
         pre_func(req)
 
     resp = agent.execute(req)
     if post_func is not None:
-        post_func(req, resp)
```

Comment:
This change lets tests tweak the valid response that is loaded from the json *_resp files. I use it to drop the name label and change the name field in the valid response.